va/va_backend.h

```diff
@@ -140,7 +140,9 @@ struct VADriverVTable
 		unsigned int size,		/* in */
 		unsigned int num_elements,	/* in */
 		void *data,			/* in */
-		VABufferID *buf_id		/* out */
+                unsigned int *unit_size,
+                unsigned int *pitch,
+                VABufferID *buf_id
```

Comment:
Please don't make this an incompatible driver API change by modifying the signature of the existing function - instead add the new function vaCreateBuffer2() in the driver API just like the external API.

Reply:
add another function pointer in the v-table , right?

Reply:
Yes, you should add another function pointer at the end of  the vtable.  your change breaks the compiling of other vaapi backend drivers

Reply:
accept